server/src/main/java/module-info.java

```diff
@@ -460,4 +460,5 @@
             org.elasticsearch.serverless.shardhealth,
             org.elasticsearch.serverless.apifiltering;
     exports org.elasticsearch.lucene.spatial;
+    exports org.elasticsearch.plugins.loading;
```

Comment:
Well this is ironic.

Reply:
Looks like I was cheating. The IDE automatically exported the new package, making me miss a bunch of cross-module access problems. I'll sort that out and move the PR out of draft when it's ready again.